### ISSUE TITLE
A11y_OSPO Production_Home Page_Page Titled: The title "Microsoft Open Source" is programmatically defined twice.

### ISSUE THREAD

**Issue by Rachita98** [microsoft/opensource.microsoft.com#119]
**GitHub Tags:**
#A11yTCS; #A11yMAS; #Win10; #DesktopWeb; #A11ySev4; #P2_WCP; #WCP; #P2_OSPOProduction_Web_Sep2021; #OSPO Production; #WCAG2.4.2; #Page Titled; #Benchmark

**Environment Details:**
URL: https://opensource.microsoft.com/

**OS and Browser Details:**
Microsoft Windows 10 Enterprise Version 21H1 (OS build 19043.1165)
Microsoft Edge Version 95.0.1000.0 (Official build) dev (64-bit)

**Repro Steps:**

1. Hit the URL:https://opensource.microsoft.com/
2. Check whether the title "Microsoft Open Source" is programmatically defined twice or not.

**Actual Result:**
The title "Microsoft Open Source" is programmatically defined twice.

**Expected Result:**
The title "Microsoft Open Source" should be programmatically defined once.

**User Impact:**
Users with disabilities will not know the purpose of the page if the title is programmatically defined twice.

**Comment by Aaron-Junker** [comment]
I will work on this issue.
